lib/ansible/plugins/doc_fragments/influxdb.py

```diff
@@ -34,6 +34,10 @@ class ModuleDocFragment(object):
     - The port on which InfluxDB server is listening
     type: int
     default: 8086
+  path:
+    description:
+    - The path on which InfluxDB server is accessible
+    type: str
```

Comment:
please append a `version_added: '2.9'` after `type: str`

Reply:
Changed as requested, (except that I've used double quotes for consistency within the file).